specs/OracleQueries.yml

```diff
@@ -468,6 +485,36 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/activeOfficerDetails'
+    managingOfficerList:
+      type: object
+      required:
+        - items_per_page
```

Comment:
we have no need to page. Just return the complete list of MOs

Reply:
We can leave it as paged but would always ask for some max (100 say). We need more testing for paged endpoints than a simple one.